Modules/Sources/NetworkingCore/Model/SalesChannelType.swift

```diff
@@ -0,0 +1,32 @@
+import Foundation
+
+public enum SalesChannelType {
+    case pointOfSale
+}
+
+extension SalesChannelType: RawRepresentable {
+    public init?(rawValue: String) {
+        switch rawValue {
+        case "pos-rest-api":
+            self = .pointOfSale
+        default:
+            return nil
+        }
+    }
+
+    public var rawValue: String {
+        switch self {
+        case .pointOfSale:
+            return description
+        }
+    }
+
+    public var description: String {
+        switch self {
+        case .pointOfSale:
+            return NSLocalizedString("saleschanneltype.pos.description",
```

Comment:
nit: or `salesChannel.pos.description` depending on the final name of the enum
```suggestion
            return NSLocalizedString("salesChannelType.pos.description",
```

Reply:
https://github.com/woocommerce/woocommerce-ios/pull/15863/commits/98aaff957f39c465f98c1d50b2dcd6748173f42f